New package: pitivi-2023030

elbachir-one opened the pull request:
<!-- Uncomment relevant sections and delete options which are not applicable -->

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

<!-- Note: If the build is likely to take more than 2 hours, please add ci skip tag as described in
https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#continuous-integration
and test at least one native build and, if supported, at least one cross build.
Ignore this section if this PR is not skipping CI.
-->

#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)

**Addresses:** #53306

elbachir-one commented:
> Works fine on x86_64-glibc. Although I do get some warnings in the UI:
> 
> ![image](https://private-user-images.githubusercontent.com/118818237/396865229-d9cdf6bb-c077-444c-9ba6-0b9e7de549b3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ2NTQwMjgsIm5iZiI6MTczNDY1MzcyOCwicGF0aCI6Ii8xMTg4MTgyMzcvMzk2ODY1MjI5LWQ5Y2RmNmJiLWMwNzctNDQ0Yy05YmE2LTBiOWU3ZGU1NDliMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMFQwMDE1MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZWQ0ZmY3MjBjNGVjODI3YWIyMDYyZDc5YTUxMmI5Mjc1MWEyNTgyYmZlNmQ5YmU5MmU5ZTZlZjliMzRkM2VmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cm9nMEUgspznX4d1Udm_XEHpCAARpwA2PyfqwC4Wjsc)
> 
> Perhaps these should be added as a dependency.

Those are some needed dependencies for certain features, but I can't find cvtracker. As for librosa, I haven't been able to make it work yet.

realcharmer commented:
I am working on packaging `librosa`, but it needs more modules which are not available, so I'll have to add them all.

elbachir-one commented:
> I am working on packaging `librosa`, but it needs more modules which are not available, so I'll have to add them all.

I have updated the `pitivi` template and added the `librosa` template. However, the `pitivi` template is still not fully tested.
